Umbraco-StarterKit/Widgets/Grid/Code/index.md

```diff
@@ -2,4 +2,21 @@
 versionFrom: 8.0.0
 ---
 
-# The Header
+# Code
+
+This allows you to add code to your grid, in such a way that the code that you add will be displayed on the frontend.
```

Comment:
I think we need to try to rephrase this, as it currently sounds as if the widgets allow the user to show the actual code on the frontend 🤔 

```suggestion
This allows you to add code to your grid, in such a way that the code that you add will be displayed on the frontend.
```

Reply:
har lavet et nyt udkast